Change 'static' to 'not changing' in the specification of GroundTruthInit (#108)
OpenSimulationInterface/osi-sensor-model-packaging: doc/spec/ground_truth_init_parameters.adoc

= Ground truth initialization parameters

All models can optionally consume `osi3::GroundTruth` via an initialization parameter called `OSMPGroundTruthInit`.

> Its purpose is to provide the model with information that is constant throughout the simulation, for example the map in OSI format or the model_reference of a MovingObject.

Reviewer: Let's discuss the wording for static/constant/moving/dynamic for changes in OSI3.8, as it also touches topics as in https://github.com/OpenSimulationInterface/open-simulation-interface/pull/768.

How about: `Its purpose is to provide the model with information that does not change throughout the simulation, for example the map in OSI format or the model_reference of a MovingObject.`

Author: I changed the sentence.

Author: I generally changed 'static'/'constant' to not 'changing'

> Its purpose is to provide the model with information that does not change throughout the simulation, for example the map in OSI format or the model_reference of a MovingObject.

Reviewer 2: ```suggestion
Its purpose is to provide the model with information in OSI format that does not change throughout the simulation, i.e. is considered static.
This can encompass, for example, the road network or referenced 3D models.
```

> `OSMPGroundTruthInit` contains all data encountered by the model that does not change during a simulation run, for example, roads.
> All data provided in this message can be assumed by the receiver to not change during the simulation run.

Reviewer 2: ```suggestion
`OSMPGroundTruthInit` contains only data encountered by the model that does not change during a simulation run.
All data provided in this message can be assumed by the receiver to be static during the simulation run.
```

* `OSMPGroundTruthInit` shall be defined as a notional discrete binary input parameter variable, with `@causality="parameter"`, `@variability="fixed"` and `@initial="exact"`.
* The MIME type of the variable shall specify the `type=GroundTruth` as part of the MIME type parameters.
* `OSMPGroundTruthInit` shall be encoded as `osi3::GroundTruth`.

> * `OSMPGroundTruthInit` shall contain all data encountered by the model that does not change during a simulation run.

Reviewer 2: ```suggestion
* `OSMPGroundTruthInit` shall contain only data encountered by the model that does not change during a simulation run.
```
